pkg/cmd/pipeline/start.go

```diff
@@ -80,11 +77,17 @@ func startCommand(p cli.Params) *cobra.Command {
 		Aliases: []string{"trigger"},
 		Short:   "Start pipelines",
 		Example: `
-# start pipeline foo by creating a pipelienrun named "foo-run-xyz123" from the namespace "bar"
+# start pipeline foo by creating a pipelinerun named "foo-run-xyz123" from the namespace "bar"
 tkn pipeline start foo --param NAME=VALUE --resource source=scaffold-git  -s ServiceAccountName  -n bar
 `,
 		SilenceUsage: true,
-		Args:         NameArg(p),
+		Args: func(cmd *cobra.Command, args []string) error {
+			initResult := flags.InitParams(p, cmd)
+			if initResult != nil {
+				return initResult
```

Comment:
We still need to call `NameArg` to get the Pipeline, don't we ?

Reply:
This is intended to return an error if `InitParams` has an error. I've renamed the variable to `err` to make this clear. `NameArg` is getting returned after this check is passed.